chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1861,9 +1890,29 @@ impl PeerManagerActor {
             return RegisterPeerResponse::InvalidNonce(Box::new(last_edge.unwrap().clone()));
         }
 
-        if msg.other_edge_info.nonce >= Edge::next_nonce(last_nonce) + EDGE_NONCE_BUMP_ALLOWED {
-            debug!(target: "network", nonce = msg.other_edge_info.nonce, last_nonce, ?EDGE_NONCE_BUMP_ALLOWED, ?self.my_peer_id, ?msg.peer_info.id, "Too large nonce");
-            return RegisterPeerResponse::Reject;
+        if msg.other_edge_info.nonce > EDGE_MIN_TIMESTAMP_NONCE {
```

Comment:
this inequality check should be a method of Edge and PartialEdgeInfo.

Reply:
for example you could have Nonce::to_utc() -> Option<time::Utc>

Reply:
done.